apollo-backend/src/main/java/io/logz/apollo/rest/RestServer.java

```diff
@@ -46,7 +46,7 @@ public RestServer(ApolloConfiguration configuration, Injector injector, UserDao
 
     @PostConstruct
     public void start() {
-        if (configuration.getSlave().isDisableApiServer()) {
+        if (configuration.getApi().isDisableApiServer()) {
             logger.info("Since I am slave, not starting the API...");
```

Comment:
Fix log to reflect that the api is disabled (not not because it's a slave)

Reply:
@igalratiner